scripts/v.dissolve/v.dissolve.py

```diff
@@ -33,36 +33,339 @@
 # %end
 # %option G_OPT_V_OUTPUT
 # %end
+# %option G_OPT_DB_COLUMN
+# % key: aggregate_column
+# % description: Name of attribute columns to get aggregate statistics for
+# % multiple: yes
+# %end
+# %option
+# % key: aggregate_method
+# % label: Aggregate statistics method
+# % description: Default is all available basic statistics
+# % multiple: yes
+# %end
+# %option G_OPT_DB_COLUMN
+# % key: stats_column
+# % description: New attribute column name for aggregate statistics results
+# % description: Defaults to aggregate column name and statistics name
+# % multiple: yes
+# %end
+# %option
+# % key: aggregate_backend
+# % label: Aggregate statistics method
+# % description: Default is all available basic statistics
+# % multiple: no
+# % required: no
+# % options: univar,sql
+# %end
+# %rules
+# % requires_all: aggregate_method,aggregate_column
+# % requires_all: stats_column,aggregate_method,aggregate_column
+# %end
+
+"""Dissolve geometries and aggregate attribute values"""
 
-import os
 import atexit
+import json
+import subprocess
 
 import grass.script as grass
```

Comment:
Yes. I wanted to avoid changing the old code, but the overall changes are so extensive that I just went ahead and changed all the old code to use the new alias, so this line is no longer there.